### PR TITLE
Add octaveless_config.yaml

### PR DESCRIPTION
**Pull request by joshuabmoore**
Add octaveless_config.yaml file which omits information theoretic SPIs with octave dependencies.

### CODE REVIEW

File: pyspi/octaveless_config.yaml [hunk 1]
```diff
@@ -0,0 +1,963 @@
...
+
+#     - estimator: symbolic
+#       k_history: 1
+#       l_history: 1
+
+#     - estimator: symbolic
+#       k_history: 10
+#       l_history: 1
+
+#   IntegratedInformation:
```
Comment (olivercliff) [inline pyspi/octaveless_config.yaml:370]: @jmoo2880 from memory only the `IntegratedInformation` SPIs would need to be commented out to get it to work without octave. See here: https://github.com/DynamicsAndNeuralSystems/pyspi/blob/a0f7167ca19f6033bb1ece836d09a80101cbf0e8/pyspi/statistics/infotheory.py#L568C26-L568C26

Comment (joshuabmoore) [reply pyspi/octaveless_config.yaml:370]: @olivercliff Ah yes, I had initially arrived at this .yaml file by sequentially removing each Information Theoretic SPI until the "Octave missing" error vanished. The motivation for this "octaveless" version was to allow users to skip the computation of SPIs with octave dependence, and continue with a reduced set of the remaining SPIs. 
Upon closer inspection, it does appear that only the IIT SPIs call on oct2py/octave, but the initialisation of the SPIs from the infotheory module will terminate when attempting to import octave (in the event it doesn't exist), rendering the remainder of the (octave independent) SPIs inaccessible. This explains why commenting out the entire info theory module (and not just IIT SPIs) works without throwing any errors relating to octave.

**Comment by olivercliff** [comment]
Ah Ok. Would it be better to move the import to inside the IIT class, or
only try important at the top if the package exists?

**Comment by joshuabmoore** [comment]
If we wanted to stick with the subset = "octaveless" idea where we use a .yaml with just the IIT SPI commented out, we could do something like a try/except block at the top of the infotheory module which will allow the rest of the classes to be available in the case that octave cannot be imported (and it throws an OSError. This seems like the most straightforward way, but would always require the user specify "octaveless" when instantiating the Calculator. Ideally, it would be great to filter out the SPIs with certain dependencies ahead of time (perhaps using additional keywords?) and give the user an idea of which SPIs will be accessible and which won't (without having to specify an "octaveless" subset for example).

**Comment by olivercliff** [comment]
I agree that it would probably be best to perhaps throw a warning but
continue running otherwise.

There are some other cases in the code where errors are caught and the
array is filled with NaNs. Could we do something like that? Send out a
warning upon instantiation of the IIT class that it won't work and then
return NaNs?

**Pull request merged**